src/main/java/io/ipfs/api/IPFS.java

```diff
@@ -490,6 +492,28 @@ public Object log() throws IOException {
         }
     }
 
+    private boolean versionIsValid(String version) {
+        String[] parts = version.split("\\.");
```

Comment:
@ianopolous I see. Your guess about the npm dependencies is indeed correct. Thanks for explaining it!